cobalt/browser/browser_module.cc

```diff
@@ -294,6 +289,28 @@ BrowserModule::BrowserModule(const GURL& url,
       current_main_web_module_timeline_id_(-1) {
   TRACE_EVENT0("cobalt::browser", "BrowserModule::BrowserModule()");
 
+  if (options.enable_on_screen_keyboard) {
+    if (OnScreenKeyboardExtensionBridge::IsSupported()) {
```

Comment:
Hi Iurii, thanks for working on this. 
Yes, it makes sense to extend the existing starboard/extension/on_screen_keyboard.h.

This if condition will break our tvOS platform, cuz we only have background color APIs enabled on the extension. We should add this if condition if only we have all osk APIs migrated for a platform.

Reply:
I've created CL to add all necessary functions to the tvOS extension (to internal code). It won't break tvOs if we merge that CL first, and then these changes.